rdt/transformers/numerical.py

```diff
@@ -157,6 +204,10 @@ def _reverse_transform(self, data):
         if self.enforce_min_max_values:
             data = data.clip(self._min_value, self._max_value)
 
+        if self.computer_representation != 'Float':
+            min_bound, max_bound = INTEGER_BOUNDS[self.computer_representation]
+            data = data.clip(min_bound, max_bound)
```

Comment:
we only need to clip once, so we could just get the strictest bounds from the parameters and clip with that right?

Reply:
I did it like this because I thought the added complexity of the code didn't justify the minimal speed gains. I'm committing the fix below, so you can compare.